src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-accent);
+}
+
+.card__product-code {
+  width: 100%;
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-secondary);
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-accent);
+}
+
+.card__price,
+.card__rating {
+  align-items: center;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-secondary);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--color-accent);
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: flex;
+  height: 40px;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: var(--color-white);
+  background-color: var(--color-blue);
+  border: 1px solid var(--color-blue);
+  transition: 0.5s;
```

Comment:
Better to set exact property that you want to animate, because if not, this transition will applied to every property